### PR TITLE
Add documentation for updating clients, people, volunteers

### PR DESCRIPTION
**Pull request by labe**
API changes: https://github.com/mes-amis/monami/pull/21225

### CODE REVIEW

File: source/includes/_clients.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ url = URI("http://app.monami.io/api/clients/")
 http = Net::HTTP.new(url.host, url.port);
 request = Net::HTTP::Post.new(url)
 request["Authorization"] = "Basic #{credential}"
-form_data = [['person', '{"first_name": "John","preferred_name": "Client","last_name": "Doe","date_of_birth": "1940-05-30","test": "test@monami.io","gender": "female","primary_phone_number": "+17075518391", "languages": "english,portuguese"}'],['address', '{"address_line1": "X Random St", "city": "San Francisco", "state": "CA", "zip": "94117"}'],['custom_fields', '{"gender_identity": "custom_value", "pronouns": "custom_value2"}']]
+form_data = [['person', '{"first_name": "Jane","preferred_name": "Client","last_name": "Doe","date_of_birth": "1940-05-30","test": "test@monami.io","gender": "female","primary_phone_number": "+17075518391", "languages": "english,portuguese"}'],['address', '{"address_line1": "X Random St", "city": "San Francisco", "state": "CA", "zip": "94117"}'],['custom_fields', '{"gender_identity": "custom_value", "pronouns": "custom_value2"}']]
```
Comment (labe) [inline source/includes/_clients.md:198]: to match the response object

File: source/includes/_clients.md [hunk 2]
```diff
@@ -284,7 +284,7 @@ This endpoint returns the newly created client.
 > POST /api/people/:person_id/clients/
 
 ```shell
-curl -i -u $MONAMI_UID:$MONAMI_SECRET https://app.monami.io/api/people/82/clients/ \
+curl -i -u $MONAMI_UID:$MONAMI_SECRET https://app.monami.io/api/people/78/clients/ \
```
Comment (labe) [inline source/includes/_clients.md:287]: to match the response object person id

File: source/includes/_people.md [hunk 3]
```diff
@@ -339,3 +339,70 @@ This endpoint returns the Person record for a given Client.
 | Parameter        | Description                                          |
 | ---------------- | ---------------------------------------------------- |
 | by_client        | The ID for the client associated with the person     |
+
+## Update a Person
+
+> PUT /api/people/:person_id
```
Comment (craigmcnamara) [inline source/includes/_people.md:345]: The Rails default is `PATCH` which allows for partial updates. `PUT` requests that the current object is updated to the exact state of the request body.

**Review by craigmcnamara — APPROVED** [review]
Docs look good, but the `PUT` calls should be `PATCH`.

**Pull request merged**